File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/accessors/PersonController.java
package io.micronaut.configuration.openapi.docs.accessors;

import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

@Controller("/persons")
class PersonController {

    @Get("/{name}")
    public Person getPerson(String name) {
        return new Person(name, 1, 2);
    }
}

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/accessors/PersonController.kt
package io.micronaut.configuration.openapi.docs.accessors

import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get

@Controller("/persons")
class PersonController {

    @Get("/{name}")
    fun getPerson(name: String): Person = Person(name, 1, 2)
}
